src/v/kafka/server/group.cc

```diff
@@ -2717,6 +2717,11 @@ ss::future<error_code> group::remove() {
               "Replicated group delete record {} at offset {}",
               _id,
               result.value().last_offset);
+        } else if (result.error() == raft::errc::shutting_down) {
+            vlog(
+              klog.warn,
+              "Cannot replicate group {} delete records due to shutdown",
+              _id);
         } else {
             vlog(
               klog.error,
```

Comment:
Could you also update this log to use the string form of the error instead of the number?  I think #6870 would have been more obvious if we were logging shutting_down instead of 19 here.